webpack.config.js

```diff
@@ -1,8 +1,9 @@
 var webpack = require('webpack');
 var development = process.env.NODE_ENV !== "production";
+var path = require('path');
 
 var PATHS = {
-  BUILD: __dirname + '/dist',
+  BUILD: path.join(__dirname, '..', 'example_site/public'),
```

Comment:
hmm. didn't know that -  not sure how I would do the sym link - might have to go over this with you to learn the best way to do it.